doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -857,6 +857,9 @@ The value of this register is copied to the |ISE| update status field of the boo
    After an update is installed or attempted, |ISE| resets the update status to ``0xFFFFFFFF`` on the next boot.
    This means that the update status is only valid for a single execution.
 
+When updating |ISE| Recovery, the status will always be ``0xFFFFFFFF``.
+The status of its update can be checked in its version field of the boot report.
```

Comment:
```suggestion
To verify that |ISE| Recovery has updated to the desired version one can check the version field of the boot report.
```

I feel we are implying that there is some kind of update_ok field in the version field of the boot report.